builds/Makefile

```diff
@@ -1429,48 +1429,53 @@ ifdef INCLUDE_DEVICE
 clean-local:: clean-llvm
 endif
 
-build-llvm32: .stamp-build-llvm .stamp-build-llvm32
-build-llvm64: .stamp-build-llvm .stamp-build-llvm32
+build-llvm32: .stamp-build-llvm32
+build-llvm64: .stamp-build-llvm64
 
-.stamp-build-llvm: $(SDK_CONFIG)
-	$(MAKE) -C $(SDK_BUILDDIR) provision-llvm-llvm64 $(SDK_ARGS)
+.stamp-compile-llvm64: $(SDK_CONFIG)
+	$(MAKE) -C $(MONO_SDK_BUILDDIR) provision-llvm-llvm64 $(SDK_ARGS)
 	$(Q) touch $@
 
-.stamp-build-llvm32: $(SDK_CONFIG)
-	$(MAKE) -C $(SDK_BUILDDIR) provision-llvm36-llvm32 $(SDK32_ARGS)
+.stamp-compile-llvm32: $(SDK_CONFIG)
+	$(MAKE) -C $(MONO_SDK_BUILDDIR) provision-llvm36-llvm32 $(SDK32_ARGS)
 	$(Q) touch $@
 
+.stamp-build-llvm64: .stamp-$(MONO_BUILD_MODE)-llvm64
```

Comment:
Where is `.stamp-download-llvm64`.

Reply:
It's handled by a pattern rule: https://github.com/xamarin/xamarin-macios/blob/a4fa62a9d28b36942c0500c9efb6c761111c17bd/builds/Makefile#L25-L26